spec/models/bear_spec.rb

```diff
@@ -1,3 +1,11 @@
+FactoryBot.define do
```

Comment:
Good work.  In the next lab that has you create factories, be sure to put them in their own files in `spec/factories`.